hack/bin/run-local.sh

```diff
@@ -5,14 +5,22 @@ set -euo pipefail
 # This script sets up the environment and runs the operator locally.
 
 NAMESPACE="monitoring"
+CERTSDIR="tmp-certs"
 declare -a OLLYOPARGS
 declare GRAFANAPORTFORWARDPID MIMIRPORTFORWARDPID ALERTMANAGERPORTFORWARDPID
 
+# Retrieves the webhook certificates from the cluster
+function getWebhookCerts {
+    mkdir -p $CERTSDIR || true
```

Comment:
`mkdir -p` always suceed, there's no error here

also I don't like very much having directories hanging around in the current directory, I would rather have this created with `mktemp`